src/app/services/dummy/dummy.service.ts

```diff
@@ -32,6 +32,10 @@ export class DummyService {
 
 		return this.http
 			.get<DummyInterface>(`${environment.assetsRoot}/dummy/dummy.json`)
-			.pipe(tap((res) => this.state.set(DUMMY_KEY, res)));
+			.pipe(
+				tap((res) =>
+					this.state.set(DUMMY_KEY, (res as unknown) as void),
```

Comment:
`as unknown) as void` is not necessary if you create the state like `makeStateKey<any>('dummyState')`. `any` should ideally be a specified interface.

Reply:
dit gaat er straks uit voor het ssr gedeelte ;)